protected/components/FilesPagination.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+Yii::import("system.web.CPagination");
+
+class FilesPagination extends CPagination {
+    public function createPageUrl($controller,$page)
+    {
+        $params=$this->params===null ? $_GET : $this->params;
+//        if($page>0) // page 0 is the default
+            $params[$this->pageVar]=$page+1;
+//        else
+//            unset($params[$this->pageVar]);
```

Comment:
If the code is commented out then it should not be present?

Reply:
The code is removed.